docs/rule/no-empty-headings.md

```diff
@@ -0,0 +1,51 @@
+# no-empty-headings
+
+Headings relay the structure of a webpage and provide a meaningful, hierarchical order of its content. If headings are empty or its text contents are inaccessible, this could confuse users or prevent them accessing sections of interest.
+
+Disallow headings (h1, h2, etc.) with no accessible text content.
+
+## Examples
+
+This rule **forbids** the following:
+
+```hbs
+<h*></h*>
+```
+
+```hbs
+<div role="heading"></div>
```

Comment:
Looks good, but let's tighten up these examples, in that `role="heading"` needs to be accompanied with `aria-level="{\d}"`. It would probably be worth checking w/screen readers to see whether `role="heading"` without `aria-level` is still identified as a heading (it may default to a specific level, IIRC). If it isn't being recognized, we are probably safe to ignore from being validated.

Reply:
VO+Safari defaults it to heading level 2. I haven't checked other screen readers yet, but I think for completeness I will add the aria-level attrib in anyway.